ironfish/src/mining/stratum/stratumServer.ts

```diff
@@ -158,6 +181,7 @@ export class StratumServer {
     this.logger.debug(`Client ${client.id} disconnected  (${this.clients.size - 1} total)`)
     this.clients.delete(client.id)
     client.close()
+    this.addToConnectionCount(client.socket.remoteAddress ?? "", -1)
```

Comment:
Suggestion related to the above change of splitting these fns out

```suggestion
    if (client.socket.remoteAddress) {
      this.removeFromConnectionCount(client.socket.remoteAddress)
    }
```